libcontainer/nsenter/nsexec.c

```diff
@@ -315,11 +314,11 @@ static int child_func(void *arg)
 	longjmp(*ca->env, ca->jmpval);
 }
 
-static int clone_parent(jmp_buf *env, int jmpval) __attribute__ ((noinline));
-static int clone_parent(jmp_buf *env, int jmpval)
+static int clone_parent(jmp_buf * env, int jmpval) __attribute__ ((noinline));
+static int clone_parent(jmp_buf * env, int jmpval)
```

Comment:
This change doesn't appear to make sense -- `jmp_buf *env` is the Linux kernel style, no? I'll have to double check why `indent` is doing that...

Reply:
According to man:

> You must use the ‘-T’ option to tell indent the name of all the typenames in your program that are defined by typedef.

Updated PR